var/spack/repos/builtin/packages/libarchive/package.py

```diff
@@ -21,15 +23,44 @@ class Libarchive(AutotoolsPackage):
     version('3.1.1', sha256='4968f9a3f2405ec7e07d5f6e78b36f21bceee6196df0a795165f89774bbbc6d8')
     version('3.1.0', sha256='64b15dfa623b323da8fc9c238b5bca962ec3b38dcdfd2ed86f5f509e578a3524')
 
-    depends_on('zlib')
-    depends_on('bzip2')
-    depends_on('lz4')
-    depends_on('xz')
-    depends_on('lzo')
-    depends_on('nettle')
-    depends_on('openssl')
-    depends_on('libxml2')
-    depends_on('expat')
+    variant('libs', default='static,shared', values=('static', 'shared'),
+            multi=True, description='What libraries to build')
+
+    # TODO: BLAKE2 and zstd (can't just add this, because libarchive is a dependency of
+    # cmake, and zstd currently uses cmake as a build system)
+    variant('compression', 'bz2lib,lz4,lzo2,lzma,zlib', values=('bz2lib', 'lz4', 'lzo2', 'lzma', 'zlib'), multi=True, description='Supported compression')
```

Comment:
Can we add default as a keyword argument:
```suggestion
    variant('compression', default='bz2lib,lz4,lzo2,lzma,zlib', values=('bz2lib', 'lz4', 'lzo2', 'lzma', 'zlib'), multi=True, description='Supported compression')
```
for better readability?